### DISCUSSION TITLE
Can I have a Github pages Blog where any user can post on to my blog?

### DISCUSSION THREAD

**Discussion by Bharathbrothers** [community/community#22460] [Other Feature Feedback, Questions, & Ideas]
<p>I wanted to have a blog where any user can post on my blog using Github Pages.</p>

**Comment by Jasius** [comment, marked as answer]
<p>GitHub pages are static so you won’t be able to do this without external resources, you could try using Netlify hosted forms for your project.</p>

**Comment by farhan-syah** [comment]
<p>You can, but you need to use your Github repo for that.</p>
<p>Start by adding collaborators to your repository or by approving pull requests from others.</p>

**Comment by aschrijver** [comment]
<p>Yes, this works rather well if your users are tech-savvy enough to be willing to interact with GH (not too hard, but needs some explaining if they are tech noobs).</p>
<p>GH has good support for Jekyll, and that could give you an easy start. As for Jekyll themes I find Minimal-Mistakes to be quite cool and well-documented.</p>
<p>Users only have to PR a Markdown doc with their post and some frontmatter fields in the top (title, author, etc.).</p>
